src/lib.rs

```diff
@@ -414,9 +414,10 @@ impl<C: Context> Secp256k1<C> {
         (bytes + word_size - 1) / word_size
     }
 
-    /// (Re)randomizes the Secp256k1 context for cheap sidechannel resistance;
-    /// see comment in libsecp256k1 commit d2275795f by Gregory Maxwell. Requires
-    /// compilation with "rand" feature.
+    /// (Re)randomizes the Secp256k1 context for cheap sidechannel resistance.
```

Comment:
In 3fa67624375ba529ad37f7786ca759224ff139fd:

Let's replace "cheap" with "extra". The cost is comparable to a signing operation.